### PR TITLE
Added necessary step to install instructions

### PR DESCRIPTION
**Pull request by andreasgriffin**
- Added necessary step to install instructions
- added numpy as python requirement

### CODE REVIEW

**Comment by doc-hex** [comment]
Looks fine, can I merge? It's still marked as draft.

File: unix/requirements.txt [hunk 1]
```diff
@@ -4,4 +4,4 @@
 #
 PySDL2
 Pillow
-
```
Comment (scgbckbone) [inline unix/requirements.txt:7]: maybe you had an install failure? `numpy` is required by `zbar-py` which is listed in `testing/requirements.txt`

File: README.md [hunk 2]
```diff
@@ -185,6 +185,8 @@ cd firmware
 # Apply address patch
 git apply unix/linux_addr.patch
 
+cd unix
```
Comment (scgbckbone) [inline README.md:188]: remove and instead change below to `pushd external/micropythohn`. Changing dir to unix here would break cmds below

**Review by scgbckbone — CHANGES_REQUESTED** [review]

File: unix/requirements.txt (hunk 1 above)
Comment (andreasgriffin) [reply unix/requirements.txt:7]: probably a mistake from the 
cd unix

Testing now.

**Comment by andreasgriffin** [comment]
Now the install instructions work flawlessly :-)

**Review by scgbckbone — APPROVED** [review]
good catch - thanks @andreasgriffin !

**Pull request merged**